packages/table/src/headers/columnHeaderCell.tsx

```diff
@@ -198,14 +209,17 @@ export class ColumnHeaderCell extends AbstractComponent<IColumnHeaderCellProps,
             pin: true,
             to: "window",
         }];
+        const classes = classNames(Classes.TABLE_TH_MENU_CONTAINER, {
+            [Classes.TABLE_TH_MENU_OPEN]: this.state.isActive,
```

Comment:
The component was _setting_ its state already, even though there was not state _defined_--check the very bottom of the file outside of this PR. Oversight from previous work on this file?